packages/@aws-cdk/aws-codepipeline-actions/lib/ecs/deploy-action.ts

```diff
@@ -58,7 +67,13 @@ export class EcsDeployAction extends Action {
       resource: props.service,
     });
 
+    const deploymentTimeout: number | undefined = props.deploymentTimeout && props.deploymentTimeout.toMinutes({ integral: true });
```

Comment:
You can make this a little bit simpler:

```suggestion
    const deploymentTimeout = props.deploymentTimeout?.toMinutes({ integral: true });
```

Reply:
So you did have the optional chaining setup in the project, I was looking at the existing code and thought it was not available. Good to know!

Reply:
Also applied it to the imageFile.fileName in the return value of `bound`